package.json

```diff
@@ -14,7 +14,7 @@
   "devDependencies": {
     "@neodrag/svelte": "^2.0.3",
     "@sveltejs/adapter-static": "^1.0.1",
-    "@sveltejs/kit": "1.3.0",
+    "@sveltejs/kit": "^1.11.0",
```

Comment:
If you update sveltekit then you will probably need to update vitepwa/sveltekit as well, does it still build properly with this change?

Reply:
It still builds properly.  I think I have a whole bunch of stuff thats been updated since polaris was made in this commit, but it doesn't seem to have any impact.

Reply:
ah no worries if something breaks I can push a hotfix for it